config/forms/2021_2022.json

```diff
@@ -1168,7 +1176,8 @@
                   "hint_text": "This is the total of rent and all charges",
                   "type": "numeric",
                   "min": 0,
-                  "step": 1
+                  "step": 1,
+                  "readonly": true
```

Comment:
Think this is missing some way of saying that the field needs to display the total of the other rent/charge fields on this page? Currently it would just be an empty non-editable field right?

Reply:
Yup! I thought that wasn't part of the ticket 😅 
Will add that